<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyARArrangeFormClass</class>
 <widget class="GUIFormBase" name="PropertyARArrangeFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PropertyARArrangeForm</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetRegistration">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Page</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date/Tme</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Position</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Additional Registation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetArrange">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ItemID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Position</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>OK/NG</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Arrangement</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpdate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyARArrange.qrc">
     <normaloff>:/Qtres/Update.png</normaloff>:/Qtres/Update.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="labelThumnail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PropertyARArrange.qrc"/>
 </resources>
 <connections/>
</ui>
